src/include/network/connection_handler_task.h

```diff
@@ -0,0 +1,78 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Terrier
+//
+// connection_handler_task.h
+//
+// Identification: src/include/network/connection_handler_task.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <event2/buffer.h>
+#include <event2/bufferevent.h>
+#include <event2/event.h>
+#include <event2/listener.h>
+
+#include <unistd.h>
+
+#include "common/container/lock_free_queue.h"
+#include "common/exception.h"
+#include "common/notifiable_task.h"
+#include "loggers/main_logger.h"
+
+namespace terrier {
```

Comment:
nit: in newer c++ you can nest namespaces in declaration, e.g. `namespace terrier::network`